xml/common_intro_available_doc_i.xml

```diff
@@ -20,134 +20,49 @@
-  <para>
-   Many commands are described in detail in their <emphasis>manual
-   pages</emphasis>. You can view manual pages by running the
-   <command>man</command> command followed by a specific command name. If the
-   <command>man</command> command is not installed on your system, install it
-   by running <command>zypper install man</command>.
-  </para>
- </note>
- <para>
-  The following documentation is available for this product:
- </para>
-
- <variablelist>
-  <varlistentry os="sles;sled">
-   <term><xref linkend="art-sle-installquick"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_installquick;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;sled">
-   <term><xref linkend="book-sle-deployment"/>
-   </term>
-   <listitem>
-    <para>
-     <remark>taroth 2016-05-20: check with cwickert if abstract is
-     still correct after the DG rewrite </remark>
-     &abstract_deployment;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="osuse">
-   <term><xref linkend="book-opensuse-startup"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_osuse_startup;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry>
-   <term><xref linkend="book-sle-admin" os="sles;sled"/><xref linkend="book-opensuse-reference" os="osuse"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_admin;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;osuse">
-   <term><xref linkend="book-virt"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_virtualization;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles">
-   <term><xref linkend="book-storage"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_storage;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;osuse">
-   <term><xref linkend="book-autoyast"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_autoyast;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry>
-   <term><xref linkend="book-security"/>
-   </term>
-   <listitem>
+<variablelist>
+ <varlistentry>
+  <term>Online documentation</term>
+  <listitem>
+   <para>
+    The online documentation for this product is available at
+<!--taroth 2021-03-18: keep the profiled links in one line,
+    moving them to separate lines would create a whitespace before
+    the full stop-->
+    <link os="sles" xlink:href="https://documentation.suse.com/#sles"/><link os="sled" xlink:href="https://documentation.suse.com/#sled"/><link os="osuse" xlink:href="http://doc.opensuse.org/"/>.
+    Browse or download the documentation in various formats.
+   </para>
+   <para os="sles;sled">
+    Find the online documentation for other products
+    at <link xlink:href="https://documentation.suse.com/"/>.</para>
+   <note>
+    <title>Latest updates</title>
     <para>
-     &abstract_security;
+     The latest documentation updates are usually available in the English version of the documentation.
```

Comment:
1. This line seems long.
2. Maybe add "only" -- or is that too negative-feeling?

Reply:
Good catch!
ad 1. I shortened the line.
ad 2. I stayed away from 'only', because otherwise we need to adjust this again in case we should move to more continuous localizations in the future .